cpplib/Tests/MainTest.cpp

```
@@ -93,6 +97,182 @@ struct FMIC_TS {
+																  xyz[0][2] + k / radius));
+				auto ret = a.CalculateEDinPoint(psoa, ElectronDensitySplines);
+				auto str = std::format("{:6.3f} {:6.3f} {:6.3f} val= {:8.3f}  {:8.3f} {:8.3f} {:8.3f}\n",
+									   a.pos_[0],
+									   a.pos_[1],
+									   a.pos_[2],
+									   ret.val,
+									   ret.grad[0],
+									   ret.grad[1],
+									   ret.grad[2]);
+				of << str;
+
```

Cppcheck (2.21.0)

[style] The function 'findCutoff' is never used.

(unusedFunction)